pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -753,18 +763,32 @@ func (lrw *logicalReplicationWriterProcessor) flushBuffer(
 	// TODO(dt): consider keeping these goroutines running for lifetime of proc
 	// rather than starting new ones for each flush.
 	chunks := make(chan []streampb.StreamEvent_KV)
+	batchSize := lrw.getBatchSize()
+
+	if lrw.skipBatching > 0 {
+		batchSize = 1
+		lrw.skipBatching--
```

Comment:
We decrement this here and then again down below on the main thread. That's fine, but I think we might benefit from a comment on what our intended backoff and then ramp up looks like.